EventVisualisation/Workflow/src/O2DPLDisplay.cxx

```diff
@@ -200,18 +200,13 @@ WorkflowSpec defineDataProcessing(ConfigContext const& cfgc)
   }
 
   bool filterTime;
-  EveWorkflowHelper::TBracket timeBracket;
+  EveWorkflowHelper::TBracket timeBracket{cfgc.options().get<float>("filter-time-min"), cfgc.options().get<float>("filter-time-max")};
 
-  if (cfgc.options().isDefault("filter-time-min") && cfgc.options().isDefault("filter-time-max")) {
+  if (timeBracket.getMin() < 0 && timeBracket.getMax() < 0) {
     filterTime = false;
-  } else if (!cfgc.options().isDefault("filter-time-min") && !cfgc.options().isDefault("filter-time-max")) {
+  } else if (timeBracket.getMin() >= 0 && timeBracket.getMax() >= 0) {
     filterTime = true;
 
```

Comment:
merging, but at the next iteration please change to 
```
filterTime = timeBracket.getMin()>=0 && timeBracket.getMax()>0;
if (timeBracket.delta()<=0) {
  throw std::runtime_error(fmt::format("Filter time bracket {} is invalid", timeBracket.asString());
}
```
Note that https://github.com/AliceO2Group/AliceO2/blob/6d7b7646f7a9ad11d2284afdcda603ccc925e1b1/EventVisualisation/Workflow/src/O2DPLDisplay.cxx#L214 is unreachable since your options has defaults.